components/style/properties/data.py

```diff
@@ -220,12 +230,19 @@ def new_style_struct(self, *args, **kwargs):
     def active_style_structs(self):
         return [s for s in self.style_structs if s.additional_methods or s.longhands]
 
+    def add_prefixed_aliases(self, property_data):
+        # FIXME Servo's DOM architecture doesn't support vendor-prefixed properties.
```

Comment:
See [CSSStyleDeclaration.webidl](https://github.com/servo/servo/blob/master/components/script/dom/webidls/CSSStyleDeclaration.webidl#L38), there are attributes for properties as their original form. For vendor-prefixed properties, that would be something like `attribute DOMString -webkit-foo;`. However, name starting with `-` is invalid in WebIDL.